models/repo_mirror.go

```diff
@@ -180,6 +181,8 @@ func (m *Mirror) runSync() bool {
 		}
 	}
 
+	cache.Remove(m.Repo.GetCommitsCountCacheKey(m.Repo.DefaultBranch, true))
```

Comment:
Does it reset default branch cache only? What about other branches cache?

Reply:
Would fix